src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -140,18 +150,51 @@ public static String getIndividualLastName(Entity entity) {
         return entity.getName().getLastName();
     }
 
-    public static String getIndividualPreferredHearingChannel(String hearingType, HearingSubtype hearingSubtype) {
-        // TODO Depends on SSCS-10273 - Needs to implement for Reference data of valid Hearing Channel codes
-        return null;
+    public static Optional<String> getIndividualPreferredHearingChannel(String hearingType,
```

Comment:
Can you change the return to a Sring

Reply:
Changed return type to String, default return is now to throw IllegalStateException as discussed over teams, we can expect only valid input to be sent in from SYA  as the prior flow would not proceed without it.